Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/view/MultiImageView.java

```diff
@@ -103,45 +105,66 @@
 
     private Context context;
     private ImageView playView;
-    private GestureDetector exoDoubleTapDetector = new GestureDetector(context, new SimpleOnGestureListener() {
-        @Override
-        public boolean onDoubleTap(MotionEvent e) {
-            callback.onDoubleTap();
-            return true;
-        }
-    });
-    private GestureDetector gifDoubleTapDetector = new GestureDetector(context, new SimpleOnGestureListener() {
```

Comment:
Uh, why is this one being removed? You're removing the pause functionality for gifs.

Reply:
Oh, damn. And the webms too. I will revert it back.

Reply:
Done.